src/adapter/src/catalog/builtin.rs

```diff
@@ -1688,6 +1688,15 @@ pub static MZ_OPERATORS: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
         .with_column("return_type_id", ScalarType::String.nullable(true)),
     is_retained_metrics_object: false,
 });
+pub static MZ_AGGREGATES: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
+    name: "mz_aggregates",
+    schema: MZ_INTERNAL_SCHEMA,
+    desc: RelationDesc::empty()
+        .with_column("oid", ScalarType::Oid.nullable(false))
+        .with_column("agg_kind", ScalarType::String.nullable(false))
+        .with_column("agg_num_direct_args", ScalarType::Int16.nullable(false)),
```

Comment:
Do we want a column for the global ID here? I'm honestly not sure. I would have thought yes, but it looks like the `mz_operators` table does not have a global ID, so it would be consistent here to leave it out. Though, I also can't remember if we assign global IDs to operators.

Reply:
I did not include global ID here since the set of aggregate functions is a subset of what exists in `mz_functions`, and global ID didn't seem to be needed. If we find it is needed later down the road we could easily add it since this new table is in `mz_internal`

Reply:
In terms of joining on `mz_functions` though I guess we would use the `oid` column for now?